Sources/ApolloCodegenLib/Templates/SwiftPackageManagerModuleTemplate.swift

```diff
@@ -34,7 +34,7 @@ struct SwiftPackageManagerModuleTemplate: TemplateRenderer {
         .library(name: "\(casedModuleName)", targets: ["\(casedModuleName)"]),
       ],
       dependencies: [
-        .package(url: "https://github.com/apollographql/apollo-ios.git", from: "1.0.0"),
+        .package(url: "https://github.com/apollographql/apollo-ios.git", from: "1.0.1"),
```

Comment:
It's debatable whether we should be including this in version updates since the dependency is `from:`.

Reply:
Yeah, I don't think we need to do this for patches, but for minor versions 100% we should. It's fine to do it, just not necessary.